manifest.json

```diff
@@ -0,0 +1,360 @@
+          "type" : "archive"
+        }
+      ]
+    },
+    {
+      "name" : "libjxl",
+      "buildsystem" : "cmake",
+      "builddir" : true,
+      "sources" : [
+        {
+          "url" : "https://github.com/libjxl/libjxl.git",
+          "commit" : "c4e0877f93506e880cd922f6c94644d79ae9adff",
+          "type" : "git"
+        }
+      ]
+    },
+    {
+      "name" : "openjpeg",
+      "buildsystem" : "cmake",
+      "builddir" : true,
+      "sources" : [
+        {
+          "url" : "https://github.com/uclouvain/openjpeg.git",
+          "commit" : "7de845f18336cc613352f497050cbb6dfdd57381",
+          "type" : "git"
+        }
+      ]
+    },
+    {
+      "name" : "libimagequant",
+      "buildsystem" : "cmake",
+      "builddir" : true,
+      "sources" : [
+        {
+          "url" : "https://github.com/ImageOptim/libimagequant.git",
+          "commit" : "9f5ecca87c1c5fb9ac9b86f301c3c7a989110a2e",
+          "type" : "git"
+        }
+      ]
+    },
+    {
+      "name" : "libspng",
+      "buildsystem" : "cmake",
+      "builddir" : true,
+      "sources" : [
+        {
+          "url" : "https://github.com/randy408/libspng.git",
+          "commit" : "81338e3ffe975bd75328fa282ef561de442da6c5",
+          "type" : "git"
+        }
+      ]
+    },
+    {
+      "name" : "libgsf",
+      "buildsystem" : "autotools",
+      "builddir" : true,
+      "sources" : [
+        {
+          "url" : "https://github.com/GNOME/libgsf.git",
+          "commit" : "14d3457f49457f925abc267f3a6964cec5a57660",
+          "type" : "git"
+        }
+      ]
+    },
+    {
+      "name" : "poppler-glib",
+      "buildsystem" : "cmake",
+      "builddir" : true,
+      "config-opts" : [
+        "-DENABLE_ZLIB=ON",
+        "-DENABLE_LIBTIFF=ON",
+        "-DENABLE_LIBPNG=ON",
+        "-DENABLE_GLIB=ON",
+        "-DENABLE_CMS='lcms2'",
+        "-DENABLE_LIBOPENJPEG='openjpeg2'",
+        "-DENABLE_DCTDECODER='libjpeg'",
+        "-DENABLE_SPLASH=OFF",
+        "-DENABLE_BOOST=OFF",
+        "-DENABLE_CPP=OFF"
+      ],
+      "sources" : [
+        {
+          "url" : "https://anongit.freedesktop.org/git/poppler/poppler.git",
+          "commit" : "4eaf52359344d2eb69a01d1a98524c9ad218e84a",
+          "type" : "git"
+        }
+      ]
+    },
+    {
+      "name" : "imath",
+      "buildsystem" : "cmake",
+      "builddir" : true,
+      "sources" : [
+        {
+          "url" : "https://github.com/AcademySoftwareFoundation/Imath.git",
+          "commit" : "6dc0820482aeb24a22f520d7b165a410589648c3",
+          "type" : "git"
+        }
+      ]
+    },
+    {
+      "name" : "openexr",
+      "buildsystem" : "cmake",
+      "builddir" : true,
+      "sources" : [
+        {
+          "url" : "https://github.com/AcademySoftwareFoundation/openexr.git",
+          "commit" : "81e3d1beffb27448f6d610ed1b1cc1c962d51011",
+          "type" : "git"
+        }
+      ]
+    },
+    {
+      "name" : "pkg-config",
```

Comment:
Yes, it's very unfortunate. 

I want openslide, but unfortunately the package lost it's maintainer in 2015 (!!):

https://openslide.org/

Due to the world moving on, it has become rather a hard package to build. You need to patch configure.ac and rerun autotools, and that means you also need a version of autotools from back then. And that means we have to build pkg-config to get all the autotools macros.

Reply:
and all of that is in the SDK

Reply:
But not autoconf 2.69 / automake 1.14, which is the combo that openslide needs (as far as I know).

Reply:
autoconf 2.69 is in the GNOME 40 runtime you were using. (2.71 in 41)

but there are more recent than 2015 so I don't see why it would be required by this.

we build pacakge much older than that.

Reply:
You're right, I removed auto* and pkg-config and it's building now. I'm sure it didn't work --- perhaps SDK41 changed something? Anyway, I've removed it now, thank you.